packages/eslint-plugin/src/rules/no-floating-promises.ts

```diff
@@ -65,7 +65,13 @@ export default util.createRule<Options, MessageId>({
           return;
         }
 
-        if (isUnhandledPromise(checker, node.expression)) {
+        let expression = node.expression;
+
+        if (expression.type === AST_NODE_TYPES.ChainExpression) {
+          expression = expression.expression;
+        }
```

Comment:
Nit: this is similar to `skipChainExpression` from `padding-line-between-statements`. You might consider making a util for it? I imagine this situation is one that will keep coming up.

Not a blocker, just a thought!